packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -101,6 +107,20 @@ export function makeVatKeeper(
     storage.set(`${vatID}.t.${id}`, JSON.stringify(msg));
   }
 
+  function vatStats() {
+    const objectCount = storage.get(`${vatID}.o.nextID`) - FIRST_OBJECT_ID;
+    const promiseCount = storage.get(`${vatID}.p.nextID`) - FIRST_PROMISE_ID;
+    const deviceCount = storage.get(`${vatID}.d.nextID`) - FIRST_DEVICE_ID;
+    const transcriptCount =
+      storage.get(`${vatID}.t.nextID`) - FIRST_TRANSCRIPT_ID;
+    return harden({
+      objectCount: Nat(Number(objectCount)),
```

Comment:
and then I think we can remove the `Number()` from these lines (although it's a good idea to retain the `Nat` check)

Reply:
done.